recipes/polap-fmlrc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "polap-fmlrc" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "4463d4a71a8a3273108a0ab2dff0a4c37a565c05af6ee17ee83dc62404294caf" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/goshng/polap/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('polap-fmlrc', max_pin="x.x") }}
+
+requirements:
+  run:
+    - python =2.7
+    - bioconda::ropebwt2 >=r187
+    - bioconda::fmlrc >=1.0.0
+    - conda-forge::parallel
+
+test:
+  commands:
+    - polap
+
+about:
+  home: https://github.com/goshng/polap
+  license: GPLv3
```

Comment:
Please use spdx identifiers

Reply:
Let me study what spdx identifiers are.

Reply:
I got it. GPL-3.0-or-later should be one of such spdx identifier for a license, right?

Reply:
I hope that now it is better. Please let me know if I need to edit the license part more. Thank you.

Reply:
Yes

Reply:
Thank you